homeassistant/components/overseerr/sensor.py

```diff
@@ -0,0 +1,107 @@
+"""Implementation of the Radarr sensor."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import OverseerrUpdateCoordinator
+
+
+@dataclass(frozen=True, kw_only=True)
+class OverseerrSensorEntityDescription(SensorEntityDescription):
+    """Entity description class for Overseerr sensors."""
+
+    value_fn: Callable[[OverseerrUpdateCoordinator], StateType]
+
+
+SENSOR_TYPES: tuple[OverseerrSensorEntityDescription, ...] = (
+    OverseerrSensorEntityDescription(
+        key="requested_movies",
+        translation_key="requested_movies",
+        icon="mdi:movie",
```

Comment:
currently most of the integrations are migrated to icon translations with an `icons.json`. I'd suggest you do that right away as well.